### PR TITLE
`Development`: Ignore build artifacts of the analysis of endpoint connections

### PR DESCRIPTION
**Pull request by Jan-Thurner**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

During the build process, build artifacts for the analysis of endpoint connections are being created. As they don't add anything to the project or a PR, they should not be pushed to Git.

### Description

The build artifacts of the analysis of endpoint connections are added to GitIgnore

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -226,3 +226,7 @@ data-exports/
 # Supporting scripts config
 ##############################
 /supporting_scripts/**/*.ini
+/supporting_scripts/analysis-of-endpoint-connections/build/scripts/analysis-of-endpoint-connections
+/supporting_scripts/analysis-of-endpoint-connections/build/scripts/analysis-of-endpoint-connections.bat
+/supporting_scripts/analysis-of-endpoint-connections/build/distributions/analysis-of-endpoint-connections-1.0-SNAPSHOT.tar
+/supporting_scripts/analysis-of-endpoint-connections/build/distributions/analysis-of-endpoint-connections-1.0-SNAPSHOT.zip
```
Comment (Hialus) [inline .gitignore:232]:
```suggestion
/supporting_scripts/analysis-of-endpoint-connections/build/**/*
```

Comment (Hialus) [inline .gitignore:232]:
```suggestion
/supporting_scripts/**/*
```

Comment (Hialus) [inline .gitignore:232]:
```suggestion
/supporting_scripts/analysis-of-endpoint-connections/build/**/*
```

**Review by JohannesStoehr — APPROVED** [review]

**Review by Strohgelaender — APPROVED** [review]

**Review by Hialus — APPROVED** [review]

**Review by sarpsahinalp — APPROVED** [review]

**Review by MarkusPaulsen — APPROVED** [review]
Change looks good to me.

**Review by egekurt123 — APPROVED** [review]
LGTM

**Review by krusche — APPROVED** [review]

**Pull request merged**